### PR TITLE
Incorporate Telemetry Application Documentation Updates

### PR DESCRIPTION
**Pull request by kb1lqc**
Simple update of the readme.md in the Applications/Telemetry folder and configuration file:

- Updated readme.md
- Telemetry configuration file updated to use uppercase text
- telemetry application updated to correctly use uppercase configuration file

Code formatting was not improved in this update.

### CODE REVIEW

File: Applications/Telemetry/readme.md
Comment (kb1lqd) [inline Applications/Telemetry/readme.md:21]: Per comment: https://github.com/FaradayRF/Faraday-Software/pull/85/files#r101904990

@kb1lqc the name/description here implies that you can connect multiple units in a single telemetry program. I don't recall this currently supported but is a future enhancement? If this is not currently supported (i.e. someone enter in "UNIT1CALL") then please state this limitation.

File: Applications/Telemetry/readme.md
Comment (kb1lqd) [inline Applications/Telemetry/readme.md]: @kb1lqc Do you want to give an example of a "Non default 5 minute timespan"? All your examples that limit to time use the default 5 minute. I'm not going to block this for review but your call.

**Review by kb1lqd — CHANGES_REQUESTED** [review]
All looks good except for the response on the telemetry.ini support for multiple units question.

File: Applications/Telemetry/readme.md
Comment (kb1lqc) [reply Applications/Telemetry/readme.md:21]: This is no different than the current implementation of APRS or Proxy. I've created [Issue #86](https://github.com/FaradayRF/Faraday-Software/issues/86) to document and address this in the future.

File: Applications/Telemetry/readme.md
Comment (kb1lqc) [reply Applications/Telemetry/readme.md]: Eventually, I think there is a better way to style this entire example of telemetry and other applications. Therefore I'll keep it basic for now and see how people learn from the current documentation and just extrapolating/experimenting. It's relatively straightforward for a developer to figure this out.

Once we get our bearings we can reassess how we do example code and update accordingly. I've created [Issue #87](https://github.com/FaradayRF/Faraday-Software/issues/87) to document and address this need.

Comment (kb1lqd) [reply Applications/Telemetry/readme.md]: From reading this I have no idea what timespan is, can I just say "...&timespan=15"? Is that 15 seconds or minutes or days?

Approving at risk, low risk for future documentation change.

**Review by kb1lqd — APPROVED** [review]

**Pull request merged**
